heron/tools/cli/src/python/response.py

```diff
@@ -33,14 +34,17 @@ class Status(object):
   Ok = 0
   InvocationError = 1
   HeronError = 100
+  DryRun = 200
 
 def status_type(status_code):
   if status_code == 0:
     return Status.Ok
   elif status_code < 100:
     return Status.InvocationError
-  else:
+  elif status_code >= 100 and status_code < 200:
```

Comment:
I think handling a range >= 100 would be a good way to future proof. If the java code adds 101 for example, the python code should pass through 101 without the need for edits.